### PR TITLE
Update of the OutputInfo interface for the sharp module

### PR DESCRIPTION
**Pull request by sinus82**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/sharp/index.d.ts#L634>>

### CODE REVIEW

**Review by fdebijl — APPROVED** [review]
Tested locally, works as expected so LGTM. Sorry for the late review, this PR got buried in my notifs.

By the way, @sinus82, you should consider adding tests for this case so you can automerge the PR yourself - something like this would work:

```ts
sharp('trim.png')
  .resize(450, 322)
  .trim()
  .toBuffer((err, data, info) => {
    // $ExpectType string
    const format = info.format;
    // $ExpectType number
    const width = info.width;
    // $ExpectType number
    const height = info.height;
    // $ExpectType number
    const trimOffsetLeft = info.trimOffsetLeft
    // $ExpectType number
    const trimOffsetTop = info.trimOffsetTop;
  });
```

File: types/sharp/index.d.ts [hunk 1]
```diff
@@ -1048,6 +1048,10 @@ declare namespace sharp {
         cropOffsetLeft?: number;
         /** Only defined when using a crop strategy */
         cropOffsetTop?: number;
+        /** Only defined when using a trim method */
+        trimOffsetLeft?: number;
+        /** Only defined when using a trim method */
+        trimOffsetTop?: number;
```
Comment (andrewbranch) [inline types/sharp/index.d.ts:1054]: I can’t find these in the actual sharp source code. Can you link to another source for this info? I don’t think a doc comment elsewhere in the file is a sufficient authority.

Comment (andrewbranch) [reply types/sharp/index.d.ts:1054]: Alternatively, @Fdebijl, when you said you tested locally, did you mean that you confirmed these properties actually exist at runtime with the latest version of sharp? Or did you just mean that you tested that the type definitions do what they intend? As long as someone can definitively prove that this is right, I’m happy.

Comment (Fdebijl) [reply types/sharp/index.d.ts:1054]: @andrewbranch I've verified the properties actually exist with sharp 0.28.0
image

Comment (andrewbranch) [reply types/sharp/index.d.ts:1054]: Perfect, thank you!

**Review by andrewbranch — APPROVED** [review]

**Comment by sinus82** [comment]
Ready to merge

**Pull request merged**